### DISCUSSION TITLE
What is the difference between the 4.31.0 and 2024-03 p2 Orbit repos?

### DISCUSSION THREAD

**Discussion by Phillipus** [eclipse-orbit/orbit-simrel#37] [Q&A] [answered]
There are two versions of the same thing:

https://download.eclipse.org/tools/orbit/simrel/maven-osgi/release/4.31.0/

and:

https://download.eclipse.org/tools/orbit/simrel/maven-osgi/2024-03/

The `4.31.0` repo says "This simple update site will be retained permanently." Maybe that's the preferred one?

Which one should I use?

Many thanks for your work on this!

**Comment by merks** [comment]
The anatomy section here describes that 

https://eclipse.dev/justj/?page=tools

Right now the later simply composes the former. 

For the coming release you should probably use 2024-06 which is updated for each milestone.

**Comment by Phillipus** [comment]
@merks Thanks for the quick reply. I'm afraid I'm none the wiser. I'm just consuming Mockito from Orbit in my `target` and Maven build. "This simple update site will be retained permanently" makes me think I should use the `4.31.0` one.

**Reply by merks** [reply]
What do you expect when there is a new version of mockito? A release site will never change. A ‘latest’ site will deliver a new version when one is available. 

I can’t answer what you should use if I don’t know what you need or expect… I also can’t clarify documentation when I don’t understand what specifically you didn’t understand.

**Comment by Phillipus** [comment]
> What do you expect when there is a new version of mockito?

I will then change the repo URL to `4.32.0` or later. I guess I'm just looking for stable and predictable update site URLs that I will manually update as required.

**Reply by merks** [reply, marked as answer]
The release sites are permanent and unchanging. The 20xx-xx sites follow nightly and milestone, until a release, and then are unchanging.   Everything else moves forward.  Pick and choose to suit your needs. 

For example, the 20xx-xx sites exist because xtext ask for a fixed url that will provide the release versions but exists before the release is created.

**Reply by Phillipus** [reply]
Thanks for the info, appreciate it. :-)
